Refactor the documentation theme and contents (#185)
rakutentech/ios-miniapp: USERGUIDE.md

Config.userDefaults?.set("MY_CUSTOM_ID", forKey: Config.Key.subscriptionKey.rawValue)
```

<a id="usage"></a>

### Usage

* [Create a MiniApp](#create-mini-app)
* [Mini App Features](#mini-app-features)
    * [Retrieving Unique ID](#retrieve-unique-id)
    * [Requesting Location Permissions](#request-location-permission)
    * [Custom Permissions](#custom-permissions)
    * [Share Mini app content](#share-mini-app-content)
    * [Retrieve User Profile details](#retrieve-user-profile-details)
* [Load the Mini App list](#load-miniapp-list)
* [Get a MiniAppInfo](#get-mini-appinfo)
* [List Downloaded Mini apps](#list-downloaded-mini-apps)
* [Advanced Features](#advanced-features)
    * [Overriding configuration on runtime](#runtime-conf)
    * [Customize history navigation](#custom-navigation)
    * [Opening external links](#Opening-external-links)
    * [Orientation Lock](#orientation-lock)

<a id="create-mini-app"></a>

### Create a MiniApp for the given `MiniAppId` :
---

> **API Docs:** [MiniApp.create](https://rakutentech.github.io/ios-miniapp/Classes/MiniApp.html), [MiniAppDisplayDelegate](https://rakutentech.github.io/ios-miniapp/Protocols/MiniAppDisplayDelegate.html)

Reviewer: In the case of iOS, you don't need to manually add the URL: Jazzy can generate a link automatically if you put a class name or method reference in backticks. For example, this should work:
```suggestion
**API Docs:** `MiniApp.create(appId:completionHandler:messageInterface:)`, `MiniAppDisplayDelegate`
```

See the Jazzy docs:
https://github.com/realm/jazzy#supported-documentation-keywords

> `MiniApp.create` is used to create a `View` for displaying a specific mini app. You must provide the mini app ID which you wish to create (you can get the mini-app ID by [Fetching Mini App Info](#fetching-mini-app-info) first). Calling `MiniApp.create` will do the following:

Reviewer: Looks like this link isn't working. I think it should be:

```suggestion
`MiniApp.create` is used to create a `View` for displaying a specific mini app. You must provide the mini app ID which you wish to create (you can get the mini-app ID by [Loading the Mini App List](#load-miniapp-list) first). Calling `MiniApp.create` will do the following:
```

}
```

<a id="user-profile-details"></a>

##### Retrieve User Profile details
---
**API Docs:** [MiniAppUserInfoDelegate](https://rakutentech.github.io/ios-miniapp/Protocols/MiniAppUserInfoDelegate.html)

Get the User profile related details using 'MiniAppMessageDelegate'.
The following delegates/interfaces will be called only if the user has allowed respective [Custom permissions](#custom-permissions)

<a id="user-profile-details-username"></a>

> #### User Name

Reviewer: I think this should be `######` so it's smaller than `Retrieve User Profile details`:

```suggestion
###### User Name
```

Retrieve user name of the User

```swift
extension ViewController: MiniAppMessageDelegate {
    func getUserName() -> String? {
        // Implementation to return the User name
        return ""
    }
}
```

<a id="user-profile-details-profilephoto"></a>

> #### Profile Photo

Reviewer: ```suggestion
###### Profile Photo
```

Retrieve Profile Photo of the User

```swift
extension ViewController: MiniAppMessageDelegate {
    func getProfilePhoto() -> String? {
        // Implementation to return the Profile photo URI
        return ""
    }
}
```

<a id="access-token-info"></a>

> #### Access Token Info

Reviewer: ```suggestion
###### Access Token Info
```
